### PR TITLE
feat: storybook related with the checkbox component

### PR DESCRIPTION
**Pull request by ardguezsoc**
- feat related with the checkbox storybook.
- feat: "defaultChecked" added to the checkbox component, now  we can have a checkbox checked when the page is loaded without any user action.

2º commit : 
- feat: storybook changes with new checkbox stories:
   - checkbox default
   - checkbox error
   - checkbox with link

### CODE REVIEW

File: src/Questions/Checkbox/__tests__/checkbox.test.js [hunk 1]
```diff
@@ -18,7 +19,7 @@ const question = {
   }
 }
 
-it('can checked/unchecked', () => {
+it('can checked/unchecked and default checked', () => {
```
Comment (ismaelocaramelo) [inline src/Questions/Checkbox/__tests__/checkbox.test.js:22]: I guess that would be a separate text for `defaultChecked` giving a question with a defaultChecked to true and also falso in the same test context

File: src/Questions/Checkbox/__tests__/checkbox.test.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const question = {
   name: 'terms_and_conditions',
   type: 'checkbox',
   isFullWidth: true,
+  defaultChecked: true,
```
Comment (ismaelocaramelo) [inline src/Questions/Checkbox/__tests__/checkbox.test.js:11]: 👍

**Review by ismaelocaramelo — CHANGES_REQUESTED** [review]
Just some minor thing :)

File: src/Questions/Checkbox/__tests__/checkbox.test.js (hunk 1 above)
Comment (ardguezsoc) [reply src/Questions/Checkbox/__tests__/checkbox.test.js]: this comment is fixed in the new commit : unit test for default checked = true and false
d406c7e

Comment (ismaelocaramelo) [reply src/Questions/Checkbox/__tests__/checkbox.test.js]: Yes, exactly!

**Review by ardguezsoc — COMMENTED** [review]
- The changes related with the json are because we added the new library to test jest-dom
- MyModal is a custom modal to show inside the checkbox-storybook it is called like that because i had the fran's modal too to check things i dont if we should keep it there or not
- Checkbox stories changes are related with the new custom story
- Checkbox /index.js  feat: to open link in a new tab or in a modal 

 @ismaelocaramelo

**Review by ismaelocaramelo — APPROVED** [review]
Nice 💥

**Pull request merged**